src/current/v25.2/create-changefeed.md

```diff
@@ -111,7 +111,7 @@ Parameter          | <div style="width:100px">Sink Type</div>      | <div style=
 Option | Value | Description
 -------|-------|------------
 <a name="avro-schema-prefix"></a>`avro_schema_prefix` | Schema prefix name | Provide a namespace for the schema of a table in addition to the default, the table name. This allows multiple databases or clusters to share the same schema registry when the same table name is present in multiple databases.<br><br>Example: `CREATE CHANGEFEED FOR foo WITH format=avro, confluent_schema_registry='registry_url', avro_schema_prefix='super'` will register subjects as `superfoo-key` and `superfoo-value` with the namespace `super`.
-<a name="compression"></a>`compression` | `gzip`, `zstd` |  Compress changefeed data files written to a [cloud storage sink]({% link {{ page.version.version }}/changefeed-sinks.md %}#cloud-storage-sink). For compression options when using a Kafka sink, see [Kafka sink configuration]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka-sink-configuration).
+<a name="compression"></a>`compression` | `gzip`, `zstd` |  Compress changefeed data files written to:<ul><li>[Cloud storage sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}#webhook-sink)</li><li><span class="version-tag">New in v25.2:</span>[Webhook sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}#cloud-storage-sink)</li></ul>. For compression options when using a Kafka sink, see [Kafka sink configuration]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka-sink-configuration).
```

Comment:
The links are reversed (webhook sink link is on cloud storage sink text and vice versa)

Reply:
Oh whoops! Thanks for catching that :)